webdriver-spec.html

```diff
@@ -2410,7 +2415,8 @@
 }</pre>
 </aside>
 
-<p>The <a>remote end steps</a> are:
+<p>The <a>remote end steps</a> <a data-lt="RES-arguments">with arguments</a> <var>url variables</var>
+ and <var>parameters</var> are:
```

Comment:
If the fact that remote end steps algorithm input is only mentioned in the processing model, one has to go through the  `remote end steps` link in the processing model to find out that this algorithm takes some input.

This is terse but makes it clearer for people like me who landed on the spec and are reading parts of it, without having an overall deep understanding.